benches/python.rs

```diff
@@ -10,25 +9,19 @@ use pyo3::Python;
 fn python_parse_numeric(bench: &mut Bencher) {
     Python::with_gil(|py| {
         bench.iter(|| {
-            // Clear PyO3 memory on each loop iteration to avoid long GC traversal overheads.
-            let _pool = unsafe { py.new_pool() };
-            black_box(python_parse(
+            python_parse(
                 py,
-                black_box(br#"  { "int": 1, "bigint": 123456789012345678901234567890, "float": 1.2}  "#),
+                br#"  { "int": 1, "bigint": 123456789012345678901234567890, "float": 1.2}  "#,
                 false,
-            ))
+            )
             .unwrap()
         });
     })
 }
 
 fn python_parse_other(bench: &mut Bencher) {
     Python::with_gil(|py| {
-        bench.iter(|| {
-            // Clear PyO3 memory on each loop iteration to avoid long GC traversal overheads.
-            let _pool = unsafe { py.new_pool() };
-            black_box(python_parse(py, black_box(br#"["string", true, false, null]"#), false)).unwrap()
-        });
+        bench.iter(|| python_parse(py, br#"["string", true, false, null]"#, false).unwrap());
```

Comment:
do we need a `black_box` around the return values?

Reply:
I couldn't see any evidence that we needed it, and it doesn't seem to have affected our benchmarks.